AUTOTEST/machine-tux-spack.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+# Copyright 1998-2019 Lawrence Livermore National Security, LLC and other
+# HYPRE Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+testname=`basename $0 .sh`
+
+# Echo usage information
+case $1 in
+   -h|-help)
+      cat <<EOF
+
+   **** This script requires certain external libraries. ****
+
+   $0 [-h|-help] {src_dir}
+
+   where: -h|-help   prints this usage information and exits
+          {src_dir}  is the hypre source directory
+          
+
+   This script runs a number of external library tests.
+
+   Example usage: $0 ../src
+
+EOF
+      exit
+      ;;
+esac
+
+# Setup
+test_dir=`pwd`
+output_dir=`pwd`/$testname.dir
+rm -fr $output_dir
+mkdir -p $output_dir
+src_dir=`cd $1; pwd`
+shift
+
+# OpenMPI limits the number of processes available by default - override
+export OMPI_MCA_rmaps_base_oversubscribe=1
+
+# Basic build and run tests
+
+# RDF - Add this later when LOBPCG residual diff issues are resolved
+#
+# spackspec="hypre@develop+debug~superlu-dist"
+# spack install $spackspec
+# spack load    $spackspec
+# spackdir=`spack location -i $spackspec`
+# test.sh basic.sh ../src -co: -mo: -spack $spackdir -eo: -spack $spackdir
+# ./renametest.sh basic $output_dir/basic-examples
+
+spackspec="hypre@develop~debug+superlu-dist"
+spack install $spackspec
+spack load    $spackspec
+spackdir=`spack location -i $spackspec`
+test.sh basic.sh ../src -co: -mo: -spack $spackdir -ro: -superlu
```

Comment:
Hi @rfalgout , is there a reason why you chose -superlu instead of -dsuperlu here? I assume we used the latter to indicate that we use super-dist (and not superlu).  Thanks.

Reply:
Hi @oseikuffuor1 .  The '-superlu' indicates the tests in 'src/test/TEST_superlu', which are actually superlu-dist tests from what I can tell.  I am just repeating what we were already running in the 'machine-tux-exlibs.sh' autotest script.  Is this what you were asking?

Reply:
@rfalgout Yes, that's what I was asking. Thanks for the clarification.